drivers/growatt/driver.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019: Rob, https://github.com/pro-sumer
+// License: GNU GPLv3, https://www.gnu.org/licenses/gpl.txt
+
+'use strict'
+
+const Homey = require('homey')
+const api = require('./api')
+
+class Growatt extends Homey.Driver {
+  onPair (socket) {
+    let username
+    let password
+    let devices = []
+
+    socket.on('login', async (data, callback) => {
+      username = data.username
```

Comment:
const { username, password } = data

Reply:
Set username and password in 1 statement.